src/commands/close-project.js

```diff
@@ -0,0 +1,50 @@
+const flow = require('lodash.flow');
+const { URL } = require('url');
+
+const github = require("../lib/github");
+const { withToken } = require('../lib/helpers/yargs/options');
+
+/**
+ * yargs builder function.
+ *
+ * @param {import('yargs').Yargs} yargs - Instance of yargs
+ */
+const builder = yargs => {
+	const baseOptions = flow([withToken]);
+
+	return baseOptions(yargs)
+		.positional('path', {
+            describe: 'URL path to project',
+            type: 'string'
+		});
+};
+
+/**
+ * Close a project.
+ *
+ * @param {object} argv - argv parsed and filtered by yargs
+ * @param {string} argv.token
+ * @param {string} argv.path
+ */
+const handler = async ({ token, path }) => {
+	const url = new URL(path);
+	const pathName = url.pathname;
+	const [ , projectType, name ] = pathName.split('/');
+
+	const { listProjects } = github({
+		personalAccessToken: token
+	});
+
+	const openProjects = await listProjects({ org: name }).catch(error => {
+		throw new Error(`Listing all open projects failed. Response: ${error}.`);
+	});
+
+	console.log(openProjects);
+};
```

Comment:
To be consistent with the existing commands, this command should implement...

- Support for the `--json` option
- Consistent output by using the `print-output` helper

See [src/commands/create-project.js](https://github.com/Financial-Times/github/blob/master/src/commands/create-project.js) for an example.

Reply:
Sounds good! I will do that 👍